src/client/js/controllers.js

```diff
@@ -93,31 +98,32 @@ netStatsApp.controller('StatsCtrl', function ($scope, $filter, $localStorage, so
 
   // Socket listeners
   // ----------------
-
-  socket.on('open', function open() {
-    socket.emit('ready');
-    console.log('The connection has been opened.');
-  })
-    .on('end', function end() {
+  socket
+    .on('connect', function open () {
+      socket.emit('ready');
+      console.log('The connection has been opened.');
+    })
+    .on('end', function end () {
       console.log('Socket connection ended.')
     })
-    .on('error', function error(err) {
+    .on('error', function error (err) {
       console.log(err);
     })
-    .on('reconnecting', function reconnecting(opts) {
+    .on('reconnecting', function reconnecting (opts) {
       console.log('We are scheduling a reconnect operation', opts);
     })
-    .on('data', function incoming(data) {
+    .on('b', function (data) {
```

Comment:
is this correct?

Reply:
yes, socket.io does not have a data event anymore. So you have to choose an name. It was `broadcast` before and i did shortcut this to `b` because it is less bytes in the response.